### ISSUE TITLE
Cant log into worls

### ISSUE THREAD

**Issue by DarGo68** [FTBTeam/FTB-Modpack-Issues#2636]
### Modpack

FTB Legend of the Eyes

### Modpack version

1.2.0

### Describe the bug

I pay for a minecraft server with G-Portal. The server was changed to FTB Legend of the Eyes. The server loads up correctly and appears in the server list. When i attempt to spawn into the worls i get a split second of the world but then get kicked with the error message in the screen shot below. I have restarted both my launcher and the server bu the same error each time

### Steps to reproduce

launched ftb app and selected the Legend of the eye mod pack
Select Multiplayer
Click on the server name
Attempt to login and get kicked

### Expected behaviour

I expected to spawn into a brand new world

### Screenshots

<img width="808" alt="image" src="https://user-images.githubusercontent.com/124174420/216078786-6b97cbcd-0b67-4490-91ce-f80d372135e5.png">

**Comment by Bigtuna572** [comment]
Same problem but I've been playing on my world for a week now. After update I couldn't login anymore.

**Comment by Eacaw** [comment]
I am also experiencing the same issue, having just downloaded the server files and tried to boot it locally before uploading to a remote server to play with friends. 

Versions:
kubejs-fabric-1902.6.0-build.135
fabric-api-0.70.0+1.19.2

**Comment by TheonlyTazz** [comment]
Legend of the Eyes V1.2 should be shipped with
`kubejs-fabric-1902.6.0-build.140.jar` and `fabric-api-0.73.0+1.19.2.jar`

which is very strange that ur server files didnt do that. would u mind comparing the server modlist with the one from your client?

**Comment by Eacaw** [comment]
Client files are:
`fabric-api-0.73.0+1.19.2`
`kubejs-fabric-1902.6.0-build.140`

**Comment by TheonlyTazz** [comment]
after checking with Eacaw, it seems that the v1.1.1 Server installer got downloaded
be sure to check your Server Installer, it should say `102_2325`
if u have been using `102_2322` you were using the v1.1.1

**Comment by DarGo68** [comment]
Been busy sorry for the delay in replying.
I downgraded my client and the server to 1.11.1 and was able to open a local world. I upogaded the server and my client back to 1.12.0 and am now able to join my server.

just for the info the server was using fabric 0.70.0+1.19.2 and kubejs 1902.6.0 Build 135 when it was on version 1.11.1
Now its on 1.12.0 its using fabric 0.73.0+1.19.2 and kubejs 1902.6.0 Build 140

G-portal server doesnt require me to upload the files themselves so im not sure where they get them from so dont know if it was 102_2322 but its all working now.
Thanks for the help
